src/HttpBasicAuthentication.php

```diff
@@ -140,8 +140,9 @@ public function __invoke(RequestInterface $request, ResponseInterface $response,
                     ->withHeader("WWW-Authenticate", sprintf('Basic realm="%s"', $this->options["realm"]));
 
                 return $this->error($request, $response, [
-                    "message" => "Callback returned false"
-                ]);
+                    "message" => "Callback returned false",
+                    "user" => $user,
+                    ]);
```

Comment:
Indentation seems to be bit wrong here.

Reply:
It doesn't work on my vs code for some reason.

Reply:
Ok, I can fix it locally while merging.